pyproject.toml

```diff
@@ -1,6 +1,45 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "rdfhash"
+version = "0.4.7"
+description = "De-duplicate RDF triples w/ a SPARQL query. Subjects taken from SELECT are replaced by the hash of their triples '{predicate} {object}.\n' pairs sorted."
+readme = "README.md"
+license = {file = "LICENSE.txt"}
+authors = [
+    {name = "Neil Graham", email = "grahamneiln@gmail.com"}
+]
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "rdflib>=6.1.1",
+    "oxrdflib>=0.3.4",
+    "pyoxigraph>=0.3.16",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest>=7.1.2",
+]
+
+[project.urls]
+Homepage = "https://github.com/NeilGraham/rdfhash"
```

Comment:
Should these not point to this copy of the repo, as opposed to your private copy?